web3.go

```diff
@@ -70,6 +71,30 @@ func WeiAsGwei(w *big.Int) string {
 	return new(big.Rat).SetFrac(w, weiPerGwei).FloatString(9)
 }
 
+// IntAsFloat converts a *big.Int (ie: wei), to *big.Float (ie: ETH)
+func IntAsFloat(i *big.Int, decimals int) *big.Float {
+	f := new(big.Float)
+	f.SetPrec(100)
+	f.SetInt(i)
+	f.Quo(f, big.NewFloat(math.Pow10(decimals)))
+	return f
+}
+
+// FloatAsInt converts a float to a *big.Int based on the decimals passed in
+func FloatAsInt(amountF *big.Float, decimals int) *big.Int {
+	bigval := new(big.Float)
+	bigval.SetPrec(100)
+	bigval.SetString(amountF.String()) // have to do this to not lose precision
+
+	coinDecimals := new(big.Float)
+	coinDecimals.SetFloat64(math.Pow10(decimals))
+	bigval.Mul(bigval, coinDecimals)
+
+	amountI := new(big.Int)
+	bigval.Int(amountI) // big.NewInt(int64(amountInWeiF)) // amountInGo.Mul(amountInGo, big.NewInt(int64(math.Pow10(18))))
```

Comment:
Could sanity check the `Accuracy` return value here.

Reply:
```suggestion
        // todo: could sanity check the accuracy here
	bigval.Int(amountI) // big.NewInt(int64(amountInWeiF)) // amountInGo.Mul(amountInGo, big.NewInt(int64(math.Pow10(18))))
```